src/mixins/README.md

```diff
@@ -0,0 +1,110 @@
+# Responsive Typography Mixins
+
+This SCSS module provides mixins and utilities for applying responsive typography styles defined in a centralized `$typography` map. It supports responsive `font-size`, `line-height` and consistent `font-family` and `font-weight` settings.
+
+## Setup
+
+Before using the mixins, ensure the `$typography` map is properly configured and passed into the module:
+
+```scss
+@use '@your-design-system/mixins-typography/mixins' with (
+  $typography: $your-typography-map
+);
+```
+
+## Exports
+
+- `@mixin responsive-font($token)`
+
+## Typography Map Structure
+
+```scss
+$typography: (
+  font-family: 'Open Sans, sans-serif',
+
+  heading1: (
+    mobile: (
+      size: var(--fs-6xl),
+      line-height: var(--lh-3xl),
+    ),
+    tablet: (
+      size: var(--fs-7xl),
+      line-height: var(--lh-6xl),
+    ),
+    desktop: (
+      size: var(--fs-8xl),
+      line-height: var(--lh-8xl),
+    ),
+    font-weight: 300,
+    letter-spacing-compact: -0.01em,
+  ),
+
+  body: (
+    m: (
+      mobile: (
+        size: var(--fs-s),
+        line-height: var(--lh-s),
+      ),
+      tablet: (
+        size: var(--fs-m),
+        line-height: var(--lh-m),
+      ),
+      desktop: (
+        size: var(--fs-m),
+        line-height: var(--lh-m),
+      ),
+    ),
+  ),
+);
+```
+
+## Usage Examples
+
+### Using a token string
+
+```scss
+.title {
+  @include mixins.responsive-font('heading1');
```

Comment:
this syntax is specific of a certain way of importing the mixins, not so important maybe but it would be better to also include the @use syntax in these examples

Reply:
i added the use in these syntaxes